test/asm/.gitignore

```diff
@@ -1,3 +1,2 @@
-/quote\"file.*
```

Comment:
Doesn't this file have special handling in the test runner script?

Reply:
That's what I thought, but couldn't find any other mention of it. It's version.asm that has special handling.

Reply:
The handling was removed in #1072, actually; this is a leftover.